addons/account/models/account_partial_reconcile.py

```diff
@@ -644,5 +664,43 @@ def _create_tax_cash_basis_moves(self):
             reconciliation_plan.append((counterpart_line + lines))
 
         self.env['account.move.line']._reconcile_plan(reconciliation_plan)
-
         return moves
+
+    def _get_draft_exchange_move_vals(self):
+        self.ensure_one()
+        if self.debit_move_id.currency_id == self.credit_move_id.currency_id == self.debit_move_id.company_id.currency_id:
```

Comment:
In that case, there is no exchange difference so I don't think this check is necessary